webapp/src/components/link_tooltip/link_tooltip.test.jsx

```diff
@@ -0,0 +1,43 @@
+import {getInfoAboutLink} from './link_tooltip';
+
+describe('getInfoAboutLink should work as expected', () => {
+    it('Should return the correct owner, repo, type, and number when given a valid GitLab href and hostname with all required information', () => {
+        const href = 'https://gitlab.com/mattermost/test/-/merge_requests/123';
+        const hostname = 'gitlab.com';
+        const expected = ['mattermost', 'test', '-', 'merge_requests', '123'];
+
+        const result = getInfoAboutLink(href, hostname);
+
+        expect(result).toEqual(expected);
```

Comment:
> `['mattermost', 'test', '-', 'merge_requests', '123'];`

What value does `-` provide being returned here? It seems that function should return an object rather than an array, to make the return value more useful and less cryptic